cmd/openshift-tests/openshift-tests.go

```diff
@@ -29,7 +29,7 @@ import (
 	testginkgo "github.com/openshift/origin/pkg/test/ginkgo"
 	"github.com/openshift/origin/pkg/version"
 	exutil "github.com/openshift/origin/test/extended/util"
-	"github.com/openshift/origin/test/extended/util/cloud"
+	"github.com/openshift/origin/test/extended/util/cluster"
```

Comment:
This rename is fine, you could also pull this out into its own PR to reduce churn in this one